omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingCrudResource.java

```diff
@@ -292,4 +293,29 @@ public boolean isRetirable() {
 	public boolean isVoidable() {
 		return false;
 	}
+
+	/**
+	 * This code is largely copied from the UI Framework:
+	 * org.openmrs.ui.framework.FormatterImpl#format(org.openmrs.OpenmrsMetadata, java.util.Locale)
+	 *
+	 * @param shortClassName
+	 * @param uuid
+	 * @return localization for the given metadata, from message source, in the authenticated locale
+	 */
+    protected String getLocalization(String shortClassName, String uuid) {
+		// in case this is a hibernate proxy, strip off anything after an underscore
+		// ie: EncounterType_$$_javassist_26 needs to be converted to EncounterType
+		int underscoreIndex = shortClassName.indexOf("_$");
+		if (underscoreIndex > 0) {
+			shortClassName = shortClassName.substring(0, underscoreIndex);
+		}
+
+		String code = "ui.i18n." + shortClassName + ".name." + uuid;
+		String localization = Context.getMessageSourceService().getMessage(code);
+		if (localization == null || localization.equals(code)) {
+			return null;
+		} else {
+			return localization;
+		}
+	}
```

Comment:
Needed to pull this up to the superclass since Concept only extends DelegatingCrudResource, not MetadataDelegatingCrudResource for some reason.